internal/k8s/client.go

```diff
@@ -139,8 +137,8 @@ type HelmKubeClient interface {
 }
 
 type K8sClient struct {
-	env               Env
-	kubectlRunner     kubectlRunner
+	env Env
+	//	kubectlRunner     kubectlRunner
```

Comment:
did you mean to delete this once you got it working?

Reply:
oops, yep